### DISCUSSION TITLE
How do I get the number of cells around each node

### DISCUSSION THREAD

**Discussion by zyhsjtu** [Ferrite-FEM/Ferrite.jl#911] [General]
How do I get the number of cells around each node?

**Comment by KnutAM** [comment]
You can use the [topology](https://ferrite-fem.github.io/Ferrite.jl/stable/reference/grid/#Topology) features. 
```julia
grid = generate_grid(Quadrilateral, (3,3));
top = ExclusiveTopology(grid);
cells_connected_to_node_2 = top.vertex_to_cell[2]
```
Note that this only works for nodes at the vertices of the cell (i.e. not for nodes on faces / edges / internal in cells existing for higher-order geometric interpolations)

**Reply by zyhsjtu** [reply]
Thanks for your quick response!
